packages/babeld-auto-gw-mode/files/etc/hotplug.d/lime-config/babeld-auto-gw.lua

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/lua
```

Comment:
Is there any concrete use of the `lime-config` hotplug.d subdirectory?
It looks like a grouping that is not documented, and it is not clear for me if it should be there or not.

Reply:
Indeed lime-config hotplug.d  documentation is missing, it was introduced in early 2019. 
hotplug.d for lime-config is similar to having a generic `run_asset` configured with `ATCONFIG` but it is enabled just by puting the script in the directory.